hooks/event-conversion-utils.ts

```diff
@@ -0,0 +1,125 @@
+import Airtable from 'airtable'
+import { CommunityEvent, CommunityEventType, WorldLocation } from '~/store/modules/events.ts'
+
+export {
+  fetchCommunityEvents,
+  convertToCommunityEvent,
+  getName,
+  getType,
+  getImage,
+  getPlace,
+  getLocation,
+  getWebsite,
+  getDates,
+  formatDates
+}
+
+async function fetchCommunityEvents (apiKey: string, { days }): Promise<CommunityEvent[]> {
+  const events: CommunityEvent[] = []
```

Comment:
In order to be consistent with the rest of the code, I think this variable should be renamed to `communityEvents`